src/BlazorUI/Bit.BlazorUI.Extras/Components/MarkdownViewer/BitMarkdownViewer.ts

```
@@ -0,0 +1,42 @@
+            const notAppenedScripts = scripts.filter(s => !allScripts.find(as => as.endsWith(s)));
+
+            if (notAppenedScripts.length == 0) return Promise.resolve();
+
+            const promise = new Promise(async (resolve: any, reject: any) => {
+                try {
+                    for (let url of notAppenedScripts) await addScript(url);
+                    resolve();
+                } catch (e: any) {
+                    reject(e);
+                }
+            });
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)